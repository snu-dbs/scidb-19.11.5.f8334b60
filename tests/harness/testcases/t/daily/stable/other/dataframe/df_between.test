# Test between() with dataframe input.

--setup
create temp array TWO_D<v:int64>[i=0:39; j=0:39]
store( \
  redimension( \
    apply( \
      build(<v:int64>[dummy=0:29], dummy), \
      (i, dummy), \
      (j, 29 - dummy)), \
    TWO_D), \
  TWO_D)

--test

--error --code=scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_DATAFRAMES_NOT_SUPPORTED \
    "between(flatten(TWO_D), 0, 0, 2, 2)"

--cleanup
remove(TWO_D)
