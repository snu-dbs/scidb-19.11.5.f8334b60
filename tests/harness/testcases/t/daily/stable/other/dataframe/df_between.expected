Query was executed successfully

Query was executed successfully

[An error expected at this place for the query "between(flatten(TWO_D), 0, 0, 2, 2)". And it failed with error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_DATAFRAMES_NOT_SUPPORTED. Expected error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_DATAFRAMES_NOT_SUPPORTED.]

Query was executed successfully

